### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by nyko27**
image

### CODE REVIEW

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+    useEffect(() => {
+        (async () => {
+            console.log("start");
+            console.log(process.env.REACT_APP_API_URL);
+            console.log(books);
+            setBooks(await getBooks());
+
+
```
Comment (Ravenen) [inline src/pages/home/Home.jsx]: Empty lines here too.

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import { getBooks } from '../../utils/api/book_api';
+import CardItem from '../../components/card/CardItem';
+import "./Home.css";
+
+export default function Home() {
+    const [books, setBooks] = useState([]);
+
```
Comment (Ravenen) [inline src/pages/home/Home.jsx:8]: Delete one empty line, please.

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    useEffect(() => {
+        (async () => {
+            console.log("start");
+            console.log(process.env.REACT_APP_API_URL);
+            console.log(books);
```
Comment (Ravenen) [inline src/pages/home/Home.jsx]: Do you still need those logs?

File: src/utils/api/book_api.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+export async function addBook(newBook) {
+    return (await axiosInstance.post("/books", newBook)).data;
+}
+
+export async function updateBookById(id, updatedBook) {
+    return (await axiosInstance.put(`/books/${id}`, updatedBook)).data;
+}
+
+export async function deleteBookById(id) {
+    return (await axiosInstance.delete(`/books/${id}`)).data;
```
Comment (Ravenen) [inline src/utils/api/book_api.js]: Not sure about that, because we don't return any data, just the status of the operation. Won't it affect the logic?

**Review by Ravenen — COMMENTED** [review]
Great start, I really like the design, it exceeded my expectations.

**Review by Ravenen — APPROVED** [review]

**Review by Ari100telll — APPROVED** [review]

**Pull request merged**